### PR TITLE
[python3]: build python3-swsscommon for python3 support

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,10 @@
 ####################
 pyext/swsscommon.py
 pyext/swsscommon_wrap.cpp
```
Comment (qiluo-msft) [inline .gitignore:10]: Are they needed?

File: m4/python3.m4 [hunk 2]
```diff
@@ -0,0 +1,192 @@
+## ------------------------                                 -*- Autoconf -*-
```
Comment (qiluo-msft) [inline m4/python3.m4:1]: Is this autogenerated? Should we ignore it in source control?

Comment (lguohan) [reply m4/python3.m4:1]: no, this is manually added.

**Comment by lguohan** [comment]
retest this please

File: .gitignore (hunk 1 above)
Comment (lguohan) [reply .gitignore:10]: just ignore the previous artifacts.

**Pull request merged**